### ISSUE TITLE
How to measure accuracy of a regression model ?

### ISSUE THREAD

**Issue by ninenerd** [microsoft/LightGBM#3438]
import lightgbm as lgb
d_train = lgb.Dataset(x_train, label=y_train)
params = {}
params['learning_rate'] = 0.003
params['boosting_type'] = 'gbdt'
params['objective'] = 'binary'
params['metric'] = 'binary_logloss'
params['sub_feature'] = 0.5
params['num_leaves'] = 10
params['min_data'] = 50
params['max_depth'] = 10
clf = lgb.train(params, d_train, 100)

#Prediction
y_pred=clf.predict(x_test)

**Comment by StrikerRUS** [comment]
Use any regression metric from scikit-learn library, for example.
https://scikit-learn.org/stable/modules/classes.html#regression-metrics
```
from sklearn.metrics import mean_absolute_error

mean_absolute_error(y_test, y_pred)
```
